containers/unit_tests/TestOffsetView.hpp

```diff
@@ -586,6 +589,7 @@ void test_offsetview_offsets_rank1() {
       KOKKOS_LAMBDA(const int ii, int& lerrors) {
         offset_view_type ov(v, {ii});
         lerrors += (ov(3) != element({3 - ii}));
+        lerrors += (ov[3] != element({3 - ii}));
```

Comment:
It wasn't tested before (the 1D `[]` operator, so I tested it now since I actually deleted the operator and it would have picked up the one from `View`).